### ISSUE TITLE
swift 3 constraints

### ISSUE THREAD

**Issue by a75c6** [lourenco-marinho/ActionButton#30]
Received error: _SwiftValue nsli_superitem

Due to the following in ActionButton.swift based on breakpoint in installConstraints:

(line 138) let views = ["floatButton":self.floatButton, "parentView":self.parentView]

Can change to the following to fix:

let views: [String: UIView] = ["floatButton":self.floatButton, "parentView":self.parentView]

Based on:

http://stackoverflow.com/questions/39622996/swift-3-error-swiftvalue-nsli-superitem

**Comment by mahmutakyol** [comment]
You are right @namsoila. Source code is true actually. I think he didn't upgrade cocoapods version.
